lua/cmp/config/default.lua

```diff
@@ -88,7 +88,11 @@ return function()
     },
 
     view = {
-      entries = { name = 'custom', selection_order = 'top_down' },
+      entries = {
+        name = 'custom',
+        selection_order = 'top_down',
+      },
+      max_items = 100,
```

Comment:
I took it out of entries, because `entries` can also be a `string`. See:

```lua
---@alias cmp.EntriesConfig cmp.CustomEntriesConfig|cmp.NativeEntriesConfig|cmp.WildmenuEntriesConfig|string
```

I can probably still move it inside though. Let me know :)

Reply:
It was supposed to adopt the default-value if a string-value was specified.

However, on second thought, having the same setting in Custom/Native/Wild is redundant.

Semantically, I thought it should be in the performance section. What do you think?